akka-http-core/src/main/scala/akka/http/javadsl/ServerBuilder.scala

```diff
@@ -106,18 +106,17 @@ trait ServerBuilder {
    * Creates a [[akka.stream.javadsl.Source]] of [[akka.http.javadsl.IncomingConnection]] instances which represents a prospective HTTP server binding
    * on the given `endpoint`.
    *
-   * If the configured  port is 0 the resulting source can be materialized several times. Each materialization will
+   * Note that each materialization will create a new binding, so
+   *
+   *  * if the configured  port is 0 the resulting source can be materialized several times. Each materialization will
    * then be assigned a new local port by the operating system, which can then be retrieved by the materialized
    * [[akka.http.javadsl.ServerBinding]].
    *
-   * If the configured  port is non-zero subsequent materialization attempts of the produced source will immediately
+   *  * if the configured  port is non-zero subsequent materialization attempts of the produced source will immediately
    * fail, unless the first materialization has already been unbound. Unbinding can be triggered via the materialized
    * [[akka.http.javadsl.ServerBinding]].
-   *
-   * If no `port` is explicitly given (or the port value is negative) the protocol's default port will be used,
-   * which is 80 for HTTP and 443 for HTTPS.
    */
-  def bind(): Source[IncomingConnection, CompletionStage[ServerBinding]]
+  def connectionSource(): Source[IncomingConnection, CompletionStage[ServerBinding]]
```

Comment:
We also call it `Tcp.bind()` in Akka, but I agree something like this is better.

Just `connections` or `source`/`asSource` would work for me as well, this is :+1: too.

Reply:
> Just `connections` or `source`/`asSource` would work for me as well, this is +1 too.

All good suggestions, but lets keep `connectionSource` as I like the extra verbosity for that particular method.